lib/config/options/index.ts

```diff
@@ -1305,6 +1305,13 @@ const options: RenovateOptions[] = [
     type: 'string',
     default: 'rebase',
   },
+  {
+    name: 'stopUpdatingLabel',
+    description: 'Label to use to request the bot to stop updating a PR.',
+    type: 'string',
+    default: 'stop-updating',
+    supportedPlatforms: ['github'],
```

Comment:
Shouldn't this work on any platform which supports PR labels and which returns them when we call `getPr()`?

Reply:
Yes, that's right